Test for remove category post

anandB000 opened the pull request:
The new branch 37-fix/add-test-for-remove-category was created and added test case for remove post form category.

anandB000 committed:
Test for remove category post

The new branch 37-fix/add-test-for-remove-category was created and added test case for remove post form category.

anandB000 committed:
Test for remove category without overwrite

The test for remove category post without overwrite

sudar commented on tests/wp-tests/BM/Metabox/Posts/CategoryTest.php:
```diff
@@ -141,4 +141,84 @@ public function test_move_posts_from_one_cat_to_another_without_overwrite() {
+		$posts_in_cat1 = $this->get_posts_by_category( $cat1 );
+		$this->assertEquals( count( $posts_in_cat1 ), 0 );
+
+		// Assert that category 2 has two posts.
```
This comment is wrong. Make sure the comments are updated when making code changes.

There are other places where you have to update the comments as well.

anandB000 replied:
Test case comments changed

anandB000 committed:
Test case comments changed

Test case comments are changed function on 'test_remove_category_from_posts_without_overwrite' and 'test_remove_category_from_posts'.
